### PR TITLE
GoonSounds!

### PR DESCRIPTION
**Pull request by ghost**
:cl: ErikHanson
add: You can now hear people die from across the station thanks to Goonstation their sound system
add: Your farts are now more A E S T H E T I C
add: This is however still an experimental feature, please make sure to post your feedback onto our Github or the Forums!
/:cl:

- [x] Give all areas a proper `sound_environment`
- [x] Run some tests with a few players

### CODE REVIEW

**Comment by KittyKathy** [comment]
So does this mean every time someone opens a door, screams, honks and beheads a guy with a esword everyone will hear it from across the station?
Also nice meme
>## Uncomment to automatically give that admin rank to all players
>+AUTOADMIN Game Admin

**Comment by cacogen** [comment]
seems really neat if it isn't shit

**Comment by ghost** [comment]
@KittyKathy The game does not use the configurations in this repository ;-)

> So does this mean every time someone opens a door, screams, honks and beheads a guy with a esword everyone will hear it from across the station?

Correct

**Comment by j-mie** [comment]
It works, as I said on Discord though:

```
regarding goon sound
sound_environment is the key to it, we need to nail that
getting the right environment is key
```

So if we can get that right, this should be good for a test merge

File: config/config.txt [hunk 1]
```diff
@@ -297,7 +297,7 @@ ANNOUNCE_ADMIN_LOGOUT
 
 ## AUTOADMIN
 ## Uncomment to automatically give that admin rank to all players
-#AUTOADMIN Game Admin
+AUTOADMIN Game Admin
```
Comment (tortellinitony) [inline config/config.txt]: We don't use the config on here but maybe this isn't a great idea to have as default

**Review by tortellinitony — CHANGES_REQUESTED** [review]

**Review by tortellinitony — APPROVED** [review]

**Pull request merged**